lib/iris/tests/unit/fileformats/netcdf/test__load_aux_factory.py

```diff
@@ -26,8 +26,10 @@ def setUp(self):
         self.ap = mock.MagicMock(units="units")
```

Comment:
It is noted that this file conflicted when merged.  The resolution is essentially to retain the cube_parts variables in place of provides.
(Is not as a severe conflict than it initially appeared 👍  )